src/jit.rs

```diff
@@ -112,27 +112,29 @@ impl JitProgram {
                 "push rbx",
                 "push rbp",
                 "mov [{host_stack_pointer}], rsp",
-                "add QWORD PTR [{host_stack_pointer}], -8", // We will push RIP in "call r10" later
-                "mov rbx, rax",
+                "add QWORD PTR [{host_stack_pointer}], -8",
```

Comment:
Is this equivalent to `add QWORD PTR [{host_stack_pointer}], -8(%rsp)`?
`rsp` is the top of the stack, but we are offsetting outside the stack area, aren't we?

Reply:
Ah, this is Intel ASM. I was trying to understand this as AT&T ASM.